libs/SalesforceSDK/res/values/sf__strings.xml

```diff
@@ -47,6 +47,19 @@
 	<string name="sf__add_new_account">Add new account</string>
     <string name="sf__account_selector_text">Select an account to use</string>
 
+	<!-- SP status updates -->
```

Comment:
During IDP initiated flow / SP initiated flow, there is a callback methods returning an enum value. Those strings here are descriptions for each one.